platform/security/platform-security-core-api/pom.xml

```diff
@@ -197,17 +197,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.55</minimum>
```

Comment:
General comment on the changes to these values: I err on the side of caution with the floating point calculations jacoco makes and set these .02 below the coverage values I saw in my environment. The new code was almost completely covered; there is one short branch that does not currently have a test.